### PR TITLE
Add VAN to mock wire API endpoint

### PR DESCRIPTION
**Pull request by aryanbhasin**

### CODE REVIEW

File: lib/circlex/api/payments/payments.ex [hunk 1]
```diff
@@ -102,21 +102,24 @@ defmodule Circlex.Api.Payments.Payments do
 
       iex> host = Circlex.Test.start_server()
       iex> amount = %{amount: "12345.00", currency: "USD"}
-      iex> Circlex.Api.Payments.Payments.mock_wire("CIR3KX3L99", amount, host: host)
+      iex> beneficiaryBank = %{accountNumber: "1000000001"}
```
Comment (hayesgm) [inline lib/circlex/api/payments/payments.ex]: We should keep every variable in Elixir as `snake_case`. Also, why not just have the user pass in an `account_number` and not some map?

File: lib/circlex/emulator/router.ex [hunk 2]
```diff
@@ -37,6 +37,7 @@ defmodule Circlex.Emulator.Router do
 
   # Payments API
   forward("/v1/payments", to: Circlex.Emulator.Api.Payments.PaymentsApi)
+  forward("/v1/balances", to: Circlex.Emulator.Api.Core.BalancesApi)
```
Comment (hayesgm) [inline lib/circlex/emulator/router.ex]: Is this a different, unrelated fix? If so, we should make it a different PR.

File: test/circlex/emulator/api/mocks_api_test.exs [hunk 3]
```diff
@@ -13,11 +13,17 @@ defmodule Circlex.Emulator.Api.MocksApiTest do
             %{
```
Comment (hayesgm) [inline test/circlex/emulator/api/mocks_api_test.exs:13]: It's weird we return a raw map here, but I suppose it's because this is some strange endpoint that doesn't comply with a lot of other standards?

**Review by hayesgm — COMMENTED** [review]
looking good.

File: lib/circlex/emulator/router.ex (hunk 2 above)
Comment (aryanbhasin) [reply lib/circlex/emulator/router.ex]: this fixes the following error in treasury's circle_admin page:


will make a separate PR

File: test/circlex/emulator/api/mocks_api_test.exs (hunk 3 above)
Comment (aryanbhasin) [reply test/circlex/emulator/api/mocks_api_test.exs:13]: yes. hard to test this endpoint because we don't know how VANs are generated, so using a raw map
